docs/source/operations/orchestrator/deployment.rst

```diff
@@ -0,0 +1,179 @@
+******************************
+How to deploy the Orchestrator
+******************************
+
+This guide shows you how to deploy the Orchestrator component of Substra.
+
+Deployment
+==========
+
+Prerequisites
+-------------
+
+In order to deploy the Orchestrator you will need a fully configured Kubernetes cluster.
+
+Preparing your Helm values
+--------------------------
+
+The orchestrator deployment is packaged using Helm.
+You can find the complete description of values that can be used to configure the chart on `Artifact Hub <https://artifacthub.io/packages/helm/substra/orchestrator>`_.
+
+#. Add the Helm repository for Substra:
+
+   .. code-block:: bash
+
+      helm repo add substra https://substra.github.io/charts/
+
+#. Create a Helm values file named ``orchestrator-values.yaml`` with the following content:
+
+   .. code-block:: yaml
+
+      ingress:
+        enabled: true
+        hostname: HOSTNAME
+
+   | Replace ``HOSTNAME`` with the hostname of your Orchestrator.
+   | This will setup the ingress in order to make your Orchestrator accessible at the defined hostname.
+
+#. Setup your :term:`Substra channels<Channel>`.
+   In the ``orchestrator-values.yaml`` file, add the following content:
+        
+   .. code-block:: yaml
+
+      channels:
+        - name: CHANNEL-NAME
+          organizations: [ORG-NAME, ...]
+
+   | Replace ``CHANNEL-NAME`` with the name you want to give to your channel.
+   | Replace ``ORG-NAME, ...`` with the names of the :term:`Organizations<Organization>` that you want to participate in this channel.
```

Comment:
```suggestion
   | Replace ``ORG-NAME, ...`` with the names of the :term:`Organizations<Organization>` that you want to allow to participate in this channel.
```

Reply:
Do you want to allow them or do you actively want them as part of the channel ?

Reply:
I'd say you want to allow them. Participating us up to them. But it's very minor, I'll resolve the thread.